wazo_sysconfd/plugins/asterisk/http.py

```diff
@@ -38,3 +38,8 @@ def move_voicemail(
 @router.delete('/voicemails_context', status_code=200)
 def delete_voicemails_context(context: str, asterisk: Asterisk = Depends(get_asterisk)):
     asterisk.delete_voicemails_context(context)
+
+
+@router.delete('/moh', status_code=200)
```

Comment:
```suggestion
@router.delete('/moh', status_code=204)
```
A detail, but a delete that returns nothing should use 204 response.

Reply:
This is consistant with the other existing deletes : 
`@router.get('/delete_voicemail', status_code=200)`
`@router.delete('/voicemails_context', status_code=200)`